frontend/src/components/authentication/createAccount.tsx

```diff
@@ -55,6 +55,12 @@ export default function CreateAccount() {
   // create new user based on inputted credentials, after the form is validated
 
   const signUp = async (newAccount: Account) => {
+    // check if the backend is runing before calling anything else
+    const response = await fetch('/user');
```

Comment:
It looks like the fetch request isn't using the correct URL, try looking at the other requests

Reply:
changed to const response = await fetch(`${PORT}/users`); and is working